website/docs/extensions/developing/commands.md

```diff
@@ -61,6 +61,10 @@ const anotherCommand = extensionApi.commands.registerCommand('extension.anotherC
 }
 ```
 
+### Additional Resources
+
+When you add the command, it will be listed on the command palette. See the [command palette](/docs/extensions/developing/command-palette) for more information.
```

Comment:
A command can also have an enablement rule that can allow to hide the command from the command palette. It is also not mentioned in the command-palette doc linked here. Maybe we should mention it

```
        "category": "Optional category to prefix title",
        "enablement": "myProperty === myValue"
```